polaris-react/src/components/Grid/components/Cell/Cell.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../../../styles/common';
+
+.Cell {
+  --pc-row-xs: initial;
+  --pc-row-sm: var(--pc-row-xs);
+  --pc-row-md: var(--pc-row-sm);
+  --pc-row-lg: var(--pc-row-md);
+  --pc-row-xl: var(--pc-row-lg);
+  --pc-column-xs: initial;
+  --pc-column-sm: var(--pc-column-xs);
+  --pc-column-md: var(--pc-column-sm);
+  --pc-column-lg: var(--pc-column-md);
+  --pc-column-xl: var(--pc-column-lg);
```

Comment:
This code is pretty confusing. Are we setting everything to `initial`?

Reply:
Yeah it's hard to understand at first but I think it's better than doing [this](https://github.com/Shopify/polaris/pull/5966/commits/e6d59c002e75e544f36262bbbc6e416905473dfa#diff-44cc555ece4f1fd992b5d469080859d4487ca98bfe7e1d1b95283ec57340cc05R49-R54) even though that might be easier to understand?

It was @aaronccasanova 's suggestion and I liked it better than calling var() a ton of times in what is essentially just as hard to read css

Reply:
This is so you only have to set things like row, column, grid-template-areas at `xs and lg` as a minimum vs having to set them all. xs = 6 column, lg is where we switch to 12 and they cascade from a mobile up perspective

Reply:
@alex-page See this comment and associated demo of the behavior:
https://github.com/Shopify/polaris/pull/5966#discussion_r888569425

As Kyle mentioned, this enables the mobile first props strategy used throughout the component.
Before:
```jsx
<Grid columns={{xs: 4, sm: 4, md: 4, lg: 6, xl: 6}} />
```
After:
```jsx
<Grid columns={{xs: 4, lg: 6}} />
```